digdag-tests/src/test/resources/acceptance/require/parent_another_project.dig

```diff
@@ -0,0 +1,7 @@
++require_by_project_id:
+  require>: child_another_project
+  project_id: ${child_project_id}
```

Comment:
This string `${child_project_id}` seems as a template parameter, but it's just replaced in the test code, right? How about following an existing case
https://github.com/treasure-data/digdag/blob/c00384a7f96a373a458d805d5c5a8ef8459a5ad1/digdag-tests/src/test/resources/acceptance/require/child.dig#L2 ? I don't insist on it, though 😺

Reply:
Good point. I found another example which used ${} and followed it.
https://github.com/treasure-data/digdag/blob/master/digdag-tests/src/test/java/acceptance/RetryIT.java#L168-L170

I hesitated to judge which is better and don't have strong opinion.

Reply:
I changed the name of template parameters as disucssion.